### ISSUE TITLE
mani sync doesn't work with remotes

### ISSUE THREAD

**Issue by cwerner1** [alajmo/mani#84]
- [x] I have the latest version of mani
- [x] I have searched through the existing issues

## Info
- OS
  - [x] Mac OS X

- Shell
  - [x] Zsh

- Version: 0.25.0    
Commit: e88f231   
Date: 2023-06-07T22:04:23Z

## Problem / Steps to reproduce
If there are "remotes" in a mani File, then the Initial mani sync doesnt work

```
projects:
  example:
    path: .
    desc: A mani example

  pinto:
    path: frontend/pinto
    url: https://github.com/alajmo/pinto.git
    remotes:
      upstream: https://github.com/alajmo/pinto.git
```

1. Create the mani.yml file with the project content
2. run `mani sync`
3. Result "error: chdir ...    no such file or directory

**Comment by cwerner1** [comment]
Is related to https://github.com/alajmo/mani/issues/62#issuecomment-1707279677

**Comment by alajmo** [comment]
Was able to reproduce, fix incoming.

**Comment by betaboon** [comment]
not trying to pressure anyone, but:
any news here?
this seems to be the only problem holding me back from potentially choosing mani over vcspull :D

**Comment by alajmo** [comment]
The work is done, just trying to work in some other fixes and features to the same release. Should be done this week.

**Comment by alajmo** [comment]
Released now with 0.30.0
